### ISSUE TITLE
Aria label warning wrongly issued

### ISSUE THREAD

**Issue by ptbrowne** [cozy/cozy-ui#1338]
If a Modal does not have a `title` props but puts the `aria-label` prop, we
still have a warning. We should test for props['aria-label'] presence IMHO
before issueing the warning.

https://github.com/cozy/cozy-ui/blob/046769b67ebe61b5dbcf17ff7886a74b67387f72/react/Modal/index.jsx#L40
